### ISSUE TITLE
node-sass 3.9.3 cause build error?

### ISSUE THREAD

**Issue by lwz7512** [driftyco/ionic-app-scripts#281]
hi, i plan to build the project locally, but failed.. i use osx 10, node v6.5.0/npm 3.10.3...
blew is console running process:

```
[liwenzhi@~/mobile/ionic-app-scripts] $ npm install

> @ionic/app-scripts@0.0.37 prepublish /Users/liwenzhi/mobile/ionic-app-scripts
> jasmine JASMINE_CONFIG_PATH=src/spec/jasmine.config.json && npm run build

module.js:583
  return process.dlopen(module, path._makeLong(filename));
                 ^

Error: Module did not self-register.
    at Error (native)
    at Object.Module._extensions..node (module.js:583:18)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/liwenzhi/mobile/ionic-app-scripts/node_modules/node-sass/lib/index.js:23:15)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)

npm ERR! Darwin 15.6.0
npm ERR! argv "/usr/local/Cellar/node/6.5.0/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v6.5.0
npm ERR! npm  v3.10.3
npm ERR! code ELIFECYCLE
npm ERR! @ionic/app-scripts@0.0.37 prepublish: `jasmine JASMINE_CONFIG_PATH=src/spec/jasmine.config.json && npm run build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @ionic/app-scripts@0.0.37 prepublish script 'jasmine JASMINE_CONFIG_PATH=src/spec/jasmine.config.json && npm run build'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the @ionic/app-scripts package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     jasmine JASMINE_CONFIG_PATH=src/spec/jasmine.config.json && npm run build
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs @ionic/app-scripts
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls @ionic/app-scripts
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/liwenzhi/mobile/ionic-app-scripts/npm-debug.log
```

**Comment by alan-agius4** [comment]
Same thing here on Window 7, Node 6.9.1

**Comment by jgw96** [comment]
@alan-agius4 what version of `npm` are you running?

**Comment by alan-agius4** [comment]
node: 6.9.1

npm: 3.10.8

OS: Window 7 and Window 10

**Comment by alan-agius4** [comment]
npm: 3.10.8

**Comment by danbucholtz** [comment]
@alan-agius4, @lwz7512, I resolved this in `0.0.39`. Please update.

Thanks,
Dan

**Comment by lwz7512** [comment]
@danbucholtz OK! it works, thanks!
